### PR TITLE
Update "I have competed before" buttons

### PR DESCRIPTION
**Pull request by BiscuitCubed**
Here is the page with creating a div around the <ul> tags, and floating the buttons to the right and left.

on desktop
registrationdesktop

On the Galaxy S5
registration galaxys5

On the IPhone 6
registration iphone6

Note: There are a few devices where they are different heights because of the text being different lengths. Will need to find a fix for that.

### CODE REVIEW

**Comment by jfly** [comment]
Thanks for creating the PR. How does this look on mobile?

File: WcaOnRails/app/assets/stylesheets/registrations.scss [hunk 1]
```diff
@@ -25,3 +25,16 @@ table.registrations-table {
   z-index: 1000; /* show over sticky table headers */
   bottom: calc(5px + #{$footer-height-px});
 }
+
+#have-you-competed-nav>li:first-child {
+background: #bdc3c7;
```
Comment (jfly) [inline WcaOnRails/app/assets/stylesheets/registrations.scss]: Indentation looks screwy here, also aren't some of these rules redundant?

File: WcaOnRails/app/assets/stylesheets/registrations.scss [hunk 2]
```diff
@@ -25,3 +25,16 @@ table.registrations-table {
   z-index: 1000; /* show over sticky table headers */
   bottom: calc(5px + #{$footer-height-px});
 }
+
```
Comment (jfly) [inline WcaOnRails/app/assets/stylesheets/registrations.scss]: Confusingly enough, registrations.scss is for competition registrations, not WCA account registrations. users.scss is probably the best place for this.

**Comment by BiscuitCubed** [comment]
Yeah, I would need to do some changes for different screen sizes. Alternatively, could we just change what display they have, and set them to fill part of a container div? I think that would work better (I'll go mess around with it.)

**Comment by BiscuitCubed** [comment]
On the issue of repeated rules, you're right. The reason is because I added the styles for testing through chrome's developer tools, and I forgot to simplify them. In the dev tools it just adds a style attribute to each element. I'll go change that. I'll go add the changes to the users.scss.

**Comment by BiscuitCubed** [comment]
Okay. I added it to the users.scss, and removed it from registrations.scss

File: WcaOnRails/app/assets/stylesheets/users.scss [hunk 3]
```diff
@@ -19,3 +19,15 @@ input[type="radio"].center-vertically {
   vertical-align: middle;
   height: 100%;
 }
+
+#have-you-competed-nav>li{
```
Comment (jfly) [inline WcaOnRails/app/assets/stylesheets/users.scss]: It looks like these rules are now in both users.scss *and* registrations.scss.

File: WcaOnRails/app/assets/stylesheets/users.scss [hunk 4]
```diff
@@ -19,3 +19,15 @@ input[type="radio"].center-vertically {
...
+  background: #bdc3c7;
+  right: -5px;
+  width: 49.4%;
+  display: inline-block;
+}
+
+
+#have-you-competed-nav>li:first-child{
+right: 0px;
```
Comment (jfly) [inline WcaOnRails/app/assets/stylesheets/users.scss]: indentation

**Comment by jfly** [comment]
Please edit your original post and add screenshots for both desktop and mobile. Thanks!

**Comment by BiscuitCubed** [comment]
Okay. I just tried floating them in a div, and that seems to work really well, and is a lot easier to understand. I'm going to update it with that (I'll add screenshots too)

**Comment by BiscuitCubed** [comment]
Is there anything else that needs to be done with this? Do you know why it's failing the checks?

File: WcaOnRails/app/assets/stylesheets/registrations.scss [hunk 5]
```diff
@@ -19,9 +19,3 @@ table.registrations-table {
     }
   }
 }
-
-#registrations-actions {
```
Comment (jfly) [inline WcaOnRails/app/assets/stylesheets/registrations.scss]: Why did you delete this?

Comment (BiscuitCubed) [inline WcaOnRails/app/assets/stylesheets/registrations.scss]: Ahh... I'm not sure. I'll add it back

File: WcaOnRails/app/assets/stylesheets/registrations.scss [hunk 6]
```diff
@@ -24,4 +24,4 @@ table.registrations-table {
   position: fixed;
   z-index: 1000; /* show over sticky table headers */
   bottom: calc(5px + #{$footer-height-px});
-}
+ }
```
Comment (jfly) [inline WcaOnRails/app/assets/stylesheets/registrations.scss]: Looks like there's an extra space here.

**Comment by jfly** [comment]
Take a look at the output of travis, you'll should see style complaints. We have some rules about just what our CSS should look like.

File: WcaOnRails/app/views/devise/registrations/new.html.erb [hunk 7]
```diff
@@ -11,19 +11,20 @@
...
+                I have competed in a WCA competition.
+              </a>
+            </li>
+            <li role="presentation">
+              <a href="#never-competed" aria-controls="never-competed" role="tab" data-toggle="tab">
+                I have never competed in a WCA competition.
+              </a>
+            </li>
+          </ul>
+        </div
```
Comment (jfly) [inline WcaOnRails/app/views/devise/registrations/new.html.erb]: This is not a closing div =)

File: WcaOnRails/app/views/devise/registrations/new.html.erb [hunk 8]
```diff
@@ -11,19 +11,20 @@
...
-              I have competed in a WCA competition.
-            </a>
-          </li>
-          <li role="presentation">
-            <a href="#never-competed" aria-controls="never-competed" role="tab" data-toggle="tab">
-              I have never competed in a WCA competition.
-            </a>
-          </li>
-        </ul>
+        <div>
```
Comment (jfly) [inline WcaOnRails/app/views/devise/registrations/new.html.erb]: What's the point of this `<div>`?

**Comment by BiscuitCubed** [comment]
I'm not sure if it's necessary any more (it was used for positioning) I'm going to go check

**Comment by BiscuitCubed** [comment]
Yeah it's not necessary.

**Comment by jfly** [comment]
I'm going to close this. Feel free to reopen if you fix the build failures.
